src/Sylius/Behat/Context/Api/Admin/ManagingProductTaxonsContext.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use ApiPlatform\Core\Api\IriConverterInterface;
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Context\Api\Resources;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\TaxonInterface;
+use function _PHPStan_ccec86fc8\RingCentral\Psr7\str;
+
+final class ManagingProductTaxonsContext implements Context
+{
+    public function __construct(private ApiClientInterface $client, private IriConverterInterface $iriConverter)
+    {
+    }
+
+    /**
+     * @When I change that the :product product belongs to the :taxon taxon
+     */
+    public function iChangeThatTheProductBelongsToTheTaxon(ProductInterface $product, TaxonInterface $taxon): void
+    {
+        $this->client->buildUpdateRequest(Resources::PRODUCT_TAXON, (string) $product->getProductTaxons()->current()->getId());
+        $this->client->updateRequestData(['taxon' => $this->iriConverter->getIriFromItem($taxon)]);
+        $this->client->update();
```

Comment:
IMO we should add here a new product taxon with the given product and taxon instead of changing the existing one, especially that this behaviour is different than the one implemented in UI scenarios, so it can be confusing.

Reply:
But I've changed this endpoint that's why I Think it should be tested like that. I didn't change /product/ enpoint